modules/service/import-service.js

```diff
@@ -546,7 +546,7 @@ class ImportService {
         });
 
         await this.db.addDatasetMetadata(metadata);
-
+        otJsonService.prepareDatasetForExtractSigner(document);
```

Comment:
return value is never used

Reply:
This method performs in place sorting and it doesn't return a value